torch_xla/__init__.py

```diff
@@ -147,9 +147,13 @@ def _setup_tpu_vm_library_path() -> bool:
 
 
 def _prepare_to_exit():
-  _XLAC._prepare_to_exit()
-  if int(os.environ.get('PT_XLA_DEBUG', '0')):
-    _summarize_fn_tracker()
+  try:
+    _XLAC._prepare_to_exit()
+    if int(os.environ.get('PT_XLA_DEBUG', '0')):
+      _summarize_fn_tracker()
+  except Exception as e:
+    print('Caught an exception in the atexit callback: ', e, flush=True)
```

Comment:
nit: can you make this `logging.error("...", exc_info=e)`?

Reply:
updated.